Description: Disable the unit tests on Debian lenny, because they need NumPy 1.2
  which is not available.
Forwarded: not-needed
Author: Michael Hanke <michael.hanke@gmail.com>
diff --git a/debian/control b/debian/control
index e43d240..45034ca 100644
--- a/debian/control
+++ b/debian/control
@@ -3,7 +3,7 @@ Section: python
 Priority: extra
 Maintainer: NeuroDebian Team <team@neuro.debian.net>
 Uploaders: Michael Hanke <michael.hanke@gmail.com>, Yaroslav Halchenko <debian@onerussian.com>
-Build-Depends: debhelper (>= 7.2.18), python-all (>= 2.5), python-support (>= 0.6), python-numpy (>= 1.2), python-nose, python-scipy, help2man, python-sphinx
+Build-Depends: debhelper (>= 7.2.18), python-all (>= 2.5), python-support (>= 0.6), python-numpy (>= 1), python-nose, python-scipy, help2man, python-sphinx
 Standards-Version: 3.9.1
 Homepage: http://nipy.sourceforge.net/nibabel
 Vcs-Browser: http://git.debian.org/?p=pkg-exppsy/pynifti.git
diff --git a/debian/rules b/debian/rules
index e930b26..ce4a0ad 100755
--- a/debian/rules
+++ b/debian/rules
@@ -21,8 +21,8 @@ override_dh_auto_build:
 override_dh_auto_test:
 ifeq (,$(filter nocheck,$(DEB_BUILD_OPTIONS)))
 	for buildver in $(shell pyversions -vr); do \
-		echo "I: Running NiBabel unittests using python$$buildver"; \
-		$(MAKE) unittest PYTHON=python$$buildver || exit 1 ;\
+		echo "I: Disabled NiBabel unittests using python$$buildver"; \
+		#$(MAKE) unittest PYTHON=python$$buildver || exit 1 ;\
 	done
 endif
 
